docs/active-directory/core-concepts/01-active-directory-details-and-definitions.md

```diff
@@ -0,0 +1,111 @@
+the objects themselves are stored in a flat database file. In addition to its own name, **each object stores the name of
+the container directly above it in the hierarchy. That container object stores the name of its superior container,
+and so on, up to the root container. In this way, a logical structure is imposed that can be viewed by using AD DS tools
+as a tree of containers**. By virtue of a hierarchical naming system, the objects in the tree appear to be nested inside
+(contained by) other objects.”*
+
+The main components of the Active Directory logical structure are described as follows
+
+- Active Directory Forest
+    - The highest level in the hierarchy. A forest represents a collection of Directories. It is a security boundary, 
+    which gives administrators complete control and access to all the information inside both the Forest and all
+    Domains within the Forest. Multiple Forests may exist at a time.
+- Domain
+    - Domains partition the information that is stored inside the directory into smaller portions so that the
+  information can be more easily stored on various domain controllers and so that administrators have a greater
+  degree of control over replication. Data that is stored in the Active Directory is replicated throughout the forest from
+  one domain controller to another. Some data that is relevant to the entire forest is replicated to all domain
+  controllers. Other data that is relevant only to a specific domain is replicated only to domain controllers in
+  that particular domain. A good domain design makes it possible to implement an efficient replication topology.
+  This is important because it enables administrators to manage the flow of data across the network, that is, to
+  control how much data is replicated and where that replication traffic takes place.
+- Organizational Unit (OU)
+    - OU’s provide a way to group resources so that they can be managed as a single unit. This allows admins to apply
+group policies across many computers or control access of resources for many users. OUs can also be delegated to 
+specific admins, so that they only have control over their subset of resources.
+
+Other core components of an Active Directory are
+
+- DNS Support
+    - In Active Directory, DNS is the means by which directory clients locate, or discover, domain controllers. 
+      The primary components of the architecture for DNS support of Active Directory include the domain controller
+      Locator, Active Directory domain names in DNS, and Active Directory DNS objects.
+    - Every Active Directory domain has a DNS domain name (for example, [cohovineyard.com](http://cohovineyard.com/)),
+      and every domain joined computer has a DNS name (for example, 
+      [server1.cohovineyard.com](http://server1.cohovineyard.com/)). Architecturally, domains and computers are
+      represented both as objects in Active Directory and as nodes in DNS.
```

Comment:
Link to the wikipedia page about DNS. Also, I think this doesn't necessarily need a large explanation.

The fact that every domain has its own DNS domain name should be moved to docs specifically built around domains.

Don't think you need to get into the specifics that they are represented as nodes.